### PR TITLE
[ansible] update mcx ansible to config incrementally

### PR DESCRIPTION
**Pull request by Xichen96**
### Description of PR

Add option to let mcx ansible playbook to config mcx incrementally or completely new

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [x] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [ ] 202205

#### What is the motivation for this PR?
Currently mcx playbook will wipe out all configurations when deploying mcx, add an option to update incrementally.

#### How did you do it?
Add new function

#### How did you verify/test it?
Run test.

### CODE REVIEW

File: ansible/roles/mcx/tasks/reset_update.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    dest: /tmp/base_config.json
+
+- name: Backup config_db.json
+  shell: cp /etc/sonic/config_db.json /etc/sonic/config_db.json.bak
+
+- name: Generate new config_db.json with sonic-cfggen
+  shell: sonic-cfggen -H -k {{ HwSku }} -j /tmp/base_config.json --print-data > /etc/sonic/config_db.json
+
+- name: Config
+  shell: config reload -y
```
Comment (Blueve) [inline ansible/roles/mcx/tasks/reset_update.yml:21]: there is a config reload and will cause the device down in a short time

**Comment by wangxin** [comment]
@Xichen96 Your change got trailing whitespace at the end of line. Can you fix it to pass pre-commit check?
Even better, you can follow the guide to enable pre-commit to check your code before you run `git commit` to avoid the issue gets to github.

File: docs/testbed/README.testbed.MCX.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# What is MCX
+
+MCX is the combination of Mx and C0, a management devices that provides console access to network devices (DUT, fanout) and BMC access to servers (TODO).
+
+# Configuring MCX
+
+There is an ansible playbook mcx.yml under directory ansible that help you deploy your mcx. 
+
+There are deveral step. First register the mcx devices and devices under its manegement to sonic_{inventory}_devices.csv. Second mark the console links between mcx and network devices in sonic_{inventory}_console_links.csv with their baud rate. Third, generate connection graph. Lastly run mcx.yml ansible playbook. A new config_db.json will be generated and loaded and the old config_db.json will be backed up.
+
```
Comment (Blueve) [inline docs/testbed/README.testbed.MCX.md:10]: Can we have a sample usage here?

**Review by Blueve — APPROVED** [review]

**Pull request merged**